js/build.js

```diff
@@ -194,22 +210,45 @@
         refreshTimer = setTimeout(refresh, refreshTimeout);
       }
 
-      function inheritColor(inheritanceColorKey, colorsArray, colorIndex) {
-        var inheritanceColor = Fliplet.Themes.Current.get(inheritanceColorKey);
+      function getThemeColor(colorKey) {
+        return (themeValues && themeValues.hasOwnProperty(colorKey)) ? themeValues[colorKey] : Fliplet.Themes.Current.get(colorKey);
+      }
 
-        if (inheritanceColor) {
-          colorsArray[colorIndex] = inheritanceColor;
+      function getDeviceType() {
+        if (Modernizr.mobile) {
+          return '';
```

Comment:
Why doesn't this return `"Mobile"`? And what's the point of mapping the device types to capitalized names?

Reply:
The result of the functions is used to get the keys of the chart colors in the instance object. Example: 
'chartColor1' + '' = key for mobile color
'chartColor1' + 'Tablet' = key for tablet color
'chartColor1' + 'Desctop' = key for desctop color

Reply:
Thanks. I get it now.